shared/constants/gas.js

```diff
@@ -27,6 +27,9 @@ export const GAS_LIMITS = {
  * These are already declared in @metamask/controllers but importing them from
```

Comment:
I'm not sure if this comment will be true after this change. If there's no reason why it wouldn't be I can remove the TODO here and we can revisit this later.

Reply:
Nice catch